torch/fx/experimental/proxy_tensor.py

```diff
@@ -2193,9 +2193,12 @@ def _wrap_func(f: Callable[_P, R], phs: Sequence[PHBase]) -> Callable[_P, R]:
             stack.enter_context(_set_make_fx_tracer(self))
 
             assert self.fx_tracer is not None
+            wrap_key_func = wrap_key(func, args, self.fx_tracer, self.pre_dispatch)
```

Comment:
If we are tracing with non-strict, we should have already have _orig_mod populated in the ModuleKeyTracer. So i feel you don't need to poke into torch/fx/_symbolic_trace?